makeflow/combined_workflow.jx

```diff
@@ -0,0 +1,250 @@
+{
+  "define": {
+    "ARCHIVE_FILE_PATH": ARCHIVE_DIR + ARCHIVE_FILE,
```

Comment:
why "ARCHIVE_FILE_PATH"? Seems we should separate archived files from the overall pipeline?